components/core/src/clp/clo/CommandLineArguments.cpp

```diff
@@ -80,10 +80,18 @@ CommandLineArguments::parse_arguments(int argc, char const* argv[]) {
             "Ignore case distinctions in both WILDCARD STRING and the input files"
     );
 
+    po::options_description options_batch_control("Batch Controls");
+    options_batch_control.add_options()(
+            "batch-size,b",
+            po::value<uint64_t>(&m_batch_size)->value_name("SIZE")->default_value(1000),
```

Comment:
Let's move `1000` into `m_batch_size`'s declaration and then replace `1000` with `m_batch_size` here.